src/test/java/ru/testtask/converter/ProjectDTOConverterTest.java

```diff
@@ -0,0 +1,132 @@
+    }
+
+    @Before
+    public void initConverter(){
+        projectDTOConverter = new ProjectDTOConverter();
+    }
+
+    @Test
+    public void convertProjectToDTOTest(){
+
+        ProjectDTO resultProjectDTO = projectDTOConverter.convertProjectToDTO(project);
+        assertNotNull(resultProjectDTO);
+        assertEquals(project.getId(), resultProjectDTO.getId());
+        assertEquals(project.getName(), resultProjectDTO.getName());
+        assertEquals(2, resultProjectDTO.getAttrs().size());
+        for (int i = 0; i < resultProjectDTO.getAttrs().size(); i++) {
+            assertEquals(project.getAttributes().get(i).getId(), resultProjectDTO.getAttrs().get(i).getId());
+            assertEquals(project.getAttributes().get(i).getName(), resultProjectDTO.getAttrs().get(i).getName());
+        }
+    }
+
+    @Test
+    public void convertDTOtoProjectTest(){
+        Project resultProject = projectDTOConverter.convertDTOtoProject(createProjectDTO);
+        assertNotNull(resultProject);
+        assertEquals(createProjectDTO.getName(), resultProject.getName());
+        assertNotNull(resultProject.getId());
+        assertNotNull(resultProject.getGeometries());
+        assertNotNull(resultProject.getAttributes());
+    }
+
+    @Test
+    public void getDTOProjectsListTest(){
+        List<Attribute> attributes1 = new ArrayList<>();
+        attributes1.add(Attribute.builder().name("attr3").id("a3").build());
+        attributes1.add(Attribute.builder().name("attr4").id("a4").build());
+        Project project1 = Project.builder().id("b").name("prj1").attributes(attributes1)
+                .geometries(new ArrayList<>()).build();
+
+
+
+        List<Project> projects = new ArrayList<>();
+        projects.add(project);
+        projects.add(project1);
+
+        List<ProjectDTO> testList = projectDTOConverter.getDTOProjectsList(projects);
+        assertEquals(2, testList.size());
+        assertEquals(projects.get(0).getId(), testList.get(0).getId());
+        assertEquals(projects.get(0).getName(), testList.get(0).getName());
+        assertEquals(projects.get(1).getId(), testList.get(1).getId());
+        assertEquals(projects.get(1).getName(), testList.get(0).getName());
+
+        for (int i = 0; i < project.getAttributes().size(); i++) {
+            assertEquals(project.getAttributes().get(i).getId(), testList.get(0).getAttrs().get(i).getId());
+            assertEquals(project.getAttributes().get(i).getName(), testList.get(0).getAttrs().get(i).getName());
+            assertEquals(project1.getAttributes().get(i).getId(), testList.get(1).getAttrs().get(i).getId());
+            assertEquals(project1.getAttributes().get(i).getName(), testList.get(1).getAttrs().get(i).getName());
+        }
+    }
+
+    @Test
+    public void getAttrListFromDTOTest(){
+        AttrDTO attrDTO = new AttrDTO("a", "attr");
+        AttrDTO attrDTO1 = new AttrDTO("b", "attr1");
+        Attribute attribute = new Attribute("a", "attr");
```

Comment:
Please remove `attribute` and `attribute1 ` and use `attrDTO` and `attrDTO1` instead